backend/app/modules/vector_store/chunk_rag_data.py

```
@@ -32,71 +32,79 @@
+                    "verdict": verdict,
+                    "explanation": explanation,
+                    "source_link": source,
+                    "article_id": article_id,
+                },
+            })
 
+        logger.info(f"Generated {len(chunks)} chunks for storage.")
         return chunks
 
     except Exception as e:
         logger.exception(f"Failed to chunk the data: {e}")
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)